src/Helpers/constants.js

```diff
@@ -1077,3 +1076,25 @@ export const OPERATING_SYSTEM_REPORT_TITLE = 'Report by operating system version
 export const OPERATING_SYSTEM_REPORT_DESCRIPTION = 'A breakdown of vulnerabilities affecting the minor operating system versions of your RHEL systems. Intended to assist with informed upgrade decisions.';
 
 export const VULNERABILITY_EXPOSURE_BY_OS_REPORT_TITLE = 'Insights Vulnerability Exposure by OS Report';
+
+export const IOP_ENVIRONMENT_CONTEXT = {
+    isLoading: false,
+    isApplicationAccessible: true,
+    isStatusDisplayed: false,
+    isBusinessRiskDisplayed: false,
+    areExcludedSystemsDisplayed: false,
+    areSystemsAccessible: true,
+    isAdvancedReportingAccessible: false,
+    isExportEnabled: false,
+    isRemediationEnabled: false,
+    isSystemExclusionEditable: false,
+    isStatusEditable: false,
+    isBusinessRiskEditable: false,
+    isPairStatusEditable: false,
+    areCvesWithoutErrataTogglable: false,
+    updateDocumentTitle: document.title,
+    getUser: () => '',
+    on: () => {},
+    hideGlobalFilter: () => {},
+    mapGlobalFilter: () => {}
```

Comment:
I feel we better completely avoid adding the callbacks that are not expected in the IoP environment. Having `undefined` is semantically giving more sense

Reply:
From the code clarity standpoint it might be a bit clearer, but if some of these functions trigger by accident, I'd much rather nothing happened than dealing with "Uncaught TypeError: undefined is not a function" error and the page crashing.